### PR TITLE
Update rtk-station Configuration

### PR DESCRIPTION
**Pull request by oliviamiller**
### Summary
- Cleans up the configure.go files for the stations
- the accuracy thats being used to configure in uncenter is in meters, not a fixed value, so removed the requirement for it to be 1-5
- updated readme to explain more about the configuration
- Did a lot of testing with the Ucenter software to confirm everything works

### CODE REVIEW

File: correction-station-serial/configure.go [hunk 1]
```diff
@@ -165,7 +186,7 @@ func (c *configCommand) sendCommand(cls, id, msgLen int, payloadCfg []byte) erro
 		return err
 	}
 
-	// then wait to capture a byte
+	// then wait to capture ack response
```
Comment (randhid) [inline correction-station-serial/configure.go:189]: spelling?

**Review by randhid — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -6,8 +6,10 @@ This module is experimental.
 Provides functionality to set up a RTCM base station to send RTCM correction streams through serial or i2c communication
 connected to a radio or Bluetooth transmitter. 
 This can be used for when there is no network connection or there are no public or private NTRIP mount points nearby.
-The base station should be set up with a clear view of the sky for the best accuracy. In the config, set your desired accuracy (1-5)
-and the time required to configure the base station.
+The base station should be set up with a clear view of the sky for the best fix. In the config, set your desired accuracy (meters)
```
Comment (susWorld) [inline README.md:9]: Does this mean any number in meters is ok for required_accuracy?

File: README.md [hunk 3]
```diff
@@ -6,8 +6,10 @@ This module is experimental.
...
+and the time required to configure the base station. The base station will be configured to output the correction messages and will begin surveying
```
Comment (susWorld) [inline README.md]: time in seconds?

**Review by susWorld — APPROVED** [review]
LGTM, just a couple of questions.

File: README.md (hunk 3 above)
Comment (oliviamiller) [reply README.md]: will add

File: README.md (hunk 2 above)
Comment (oliviamiller) [reply README.md:9]: yes

**Pull request merged**